OpenRA.Mods.Common/Activities/FindAndDeliverResources.cs

```diff
@@ -194,7 +194,8 @@ public override bool Tick(Actor self)
 
 			// Find any harvestable resources:
 			List<CPos> path;
-			using (var search = PathSearch.Search(self.World, mobile.Locomotor, self, BlockedByActor.Stationary, loc =>
```

Comment:
We should keep this one, otherwise harvesters would potentially stall even though accessible resource patches are still available. This uses the pathfinding algorithm, but the path here isn't directly used for the actual path that the harvester takes, so isn't relevant to the issue.